R/checks.R

```diff
@@ -1672,7 +1672,9 @@ checkBadFiles <- function(package_dir){
         handleNote("malformed 'License:' field '", license, "'")
         return(invisible())
     }
-    ldb_file <- R.home(file.path("share", "licenses", "license.db"))
+    ldb_file <- file.path(R.home("share"), "licenses", "license.db")
+    if ( ! file.exists(ldb_file))
+        handleError("R license database cannot be found at ", ldb_file, ".")
```

Comment:
I'm not sure that this should be an error (something wrong with the package) but maybe a note indicating that restrictive use couldn't be determined. Also I think it needs to explicitly return from the function, otherwise the remaining code (including `read.dcf(ldb_file)` will be evaluated.

It's too bad there is not an easy way to 'mock' with RUnit, so this could be tested.